src/main/kotlin/se/uulm/snowballr/backend/service/ProjectService.kt

```diff
@@ -55,4 +93,29 @@ class ProjectService(
         projects.forEach { builder.addProjects(it.toGrpcProject()) }
         return builder.build()
     }
+
+    override suspend fun updateProject(request: ProjectOuterClass.Project.Update): ProjectOuterClass.Project {
+        val currentUser = userRepo.getUserById(GrpcContext.getUserIdFromContext())
+        val projectId = parseUUID(request.project.id, EntityType.PROJECT)
+        val project = repo.getProjectById(projectId)
+
+        val adminList = projectMemberRepo.getAllProjectAdmins(projectId)
+        if (!adminList.map { member -> member.userId }.contains(currentUser.id)) {
```

Comment:
Here you can simplify the condition again.
```suggestion
        val isProjectAdmin = projectMemberRepo.getAllProjectAdmins(projectId).any { it.userId == currentUser.id }
        if (!isProjectAdmin) {
```